src/components/ListItem.jsx

```diff
@@ -45,6 +64,20 @@ export function ListItem({ name, listPath, itemId, dateLastPurchased }) {
 		}
 	}, [isChecked, itemId, listPath]);
 
+	const calculateUrgency = (daysUntilNextPurchase) => {
+		if (daysUntilNextPurchase <= 7) {
+			return 'soon';
+		} else if ((daysUntilNextPurchase > 7) & (daysUntilNextPurchase < 30)) {
+			return 'kind of soon';
+		} else if (daysUntilNextPurchase >= 30) {
+			return 'not soon';
+		} else {
```

Comment:
Need to double check the AC here - `Inactive (60 days have passed since the last purchase)`

Reply:
I think this should work now..

Reply:
Maybe I'm interpreting this incorrectly, but I think rather than looking at the value for `daysUntilNextPurchase` for inactive items, we want to check `dateLastPurchased`.

Reply:
It's possible those values will be related, but I think the choice of wording in the AC (e.g. checking that 60 days have passed since the last purchase) is pretty clear.

Reply:
I would also recommend putting this function in `firebase.js` along with `comparePurchaseUrgency` since they are related.

Reply:
> It's possible those values will be related, but I think the choice of wording in the AC (e.g. checking that 60 days have passed since the last purchase) is pretty clear.

Thank you for pointing out the AC, we misslooked the AC. Now fixed with commit [3a8e78fef962b8f4d5835eabe9c0585e4c1352c3](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/pull/34/commits/3a8e78fef962b8f4d5835eabe9c0585e4c1352c3)

Reply:
> I would also recommend putting this function in `firebase.js` along with `comparePurchaseUrgency` since they are related.

This is a good recommendation, I think the change could make the calculateUrgency function accessible to different component. Solved with commit [44e9f7efb2c302dfa5601351b70c4b278f0847a6](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/pull/34/commits/44e9f7efb2c302dfa5601351b70c4b278f0847a6)